hudi-flink-datasource/hudi-flink/src/test/java/org/apache/hudi/utils/TestData.java

```diff
@@ -553,7 +554,12 @@ public static void writeDataAsBatch(
    * Initializes a writing pipeline with given configuration.
    */
   public static TestFunctionWrapper<RowData> getWritePipeline(String basePath, Configuration conf) throws Exception {
-    if (OptionsResolver.isAppendMode(conf)) {
+    if (OptionsResolver.isBulkInsertOperation(conf)) {
+      if (conf.getBoolean(FlinkOptions.WRITE_BULK_INSERT_SORT_INPUT)) {
+        throw new UnsupportedOperationException("Harness test does not support bulk insert sort yet");
```

Comment:
The sort is by default disabled ? Then we can remove this check.

Reply:
The sort is by default enabled.
But it needs more time to support it when implementing `BulkInsertFunctionWrapper`. And it is unrelated with this PR.
So I skip it before.

Reply:
Can we just support it?

Reply:
OK